hikari/undefined.pyi

```diff
@@ -29,9 +29,7 @@ from typing import TypeVar as __TypeVar
 from typing import Union as __Union
 
 class UndefinedType(__enum.Enum):
-    def __bool__(self) -> __Literal[False]:
-        ...
-
+    def __bool__(self) -> __Literal[False]: ...
```

Comment:
should we not also be reformatting pyi files with black?

Reply:
It should already be targetting them